### PR TITLE
Variable names improvement and add .gitignore

### PR DESCRIPTION
**Pull request by ilse-macias**
Variable names improvement and add .gitignore
For .gitignore I added .DS_store

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+.DS_Store
```
Comment (yvesmh) [inline .gitignore:2]: Thank you! 👍

File: src/problem0011.js [hunk 2]
```diff
@@ -225,14 +225,15 @@
 
   var largestHorizontal = getHorizontalMaxProduct(grid, 4);
   var largestVertical = getVerticalMaxProduct(grid, 4);
-  var largestDiagonal = getLeftToRightDiagonalMaxProduct(grid, 4);
-  var largestDiagonal2 = getRightToLeftDiagonalMaxProduct(grid, 4);
```
Comment (yvesmh) [inline src/problem0011.js:229]: Thanks, this variable name was not very good :)

File: src/problem0011.js [hunk 3]
```diff
@@ -225,14 +225,15 @@
...
   console.log("Largest horizontal: " + largestHorizontal);
   console.log("Largest vertical: " + largestVertical);
-  console.log("Largest diagonal (left to right): " + largestDiagonal);
-  console.log("Largest diagonal (right to left): " + largestDiagonal2);
+  console.log("Largest diagonal (left to right): " + largestDiagonalLeftToRight);
+  console.log("Largest diagonal (right to left): " + largestDiagonalRightToLeft);
+  var all = [largestHorizontal, largestVertical, largestDiagonalLeftToRight, largestDiagonalRightToLeft];
 
-  var largestOfAll = Math.max.apply(Math, [largestHorizontal, largestVertical, largestDiagonal, largestDiagonal2]);
+  var largestOfAll = Math.max.apply(Math, all);
```
Comment (yvesmh) [inline src/problem0011.js:236]: This helps read the line in small monitors easier!

File: src/problem0011.js [hunk 4]
```diff
@@ -225,14 +225,15 @@
...
-  var largestDiagonal2 = getRightToLeftDiagonalMaxProduct(grid, 4);
+  var largestDiagonalLeftToRight = getLeftToRightDiagonalMaxProduct(grid, 4);
+  var largestDiagonalRightToLeft = getRightToLeftDiagonalMaxProduct(grid, 4);
   console.log("Largest horizontal: " + largestHorizontal);
   console.log("Largest vertical: " + largestVertical);
-  console.log("Largest diagonal (left to right): " + largestDiagonal);
-  console.log("Largest diagonal (right to left): " + largestDiagonal2);
+  console.log("Largest diagonal (left to right): " + largestDiagonalLeftToRight);
+  console.log("Largest diagonal (right to left): " + largestDiagonalRightToLeft);
+  var all = [largestHorizontal, largestVertical, largestDiagonalLeftToRight, largestDiagonalRightToLeft];
```
Comment (yvesmh) [inline src/problem0011.js:234]: Minor nitpick: This line would be easier to read if it was split into multiple lines. Example:

```js
   var all = [
    largestHorizontal, 
    largestVertical, 
    largestDiagonalLeftToRight, 
    largestDiagonalRightToLeft];
```

For now, this is good, no need to change anything.

**Review by yvesmh — APPROVED** [review]

File: src/problem0011.js (hunk 4 above)
Comment (ilse-macias) [reply src/problem0011.js:234]: Thank you, I'll do that on my next PR.

**Pull request merged**